roles/kubevirt-ssp/tasks/deprovision.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: Check that common-templates.yaml still exists in /tmp
+  stat:
+    path: "/tmp/common-templates.yaml"
+  register: common_templates
+
+- name: Get Common Templates from {{ common_templates_manifest_url }}
+  block:
+  - name: Check for common-templates manifest in /tmp
+    stat:
+      path: "/tmp/common-templates.yaml"
+    register: downloaded_templates
+
+  - name: Download common-templates manifest
+    get_url:
+      url: "{{ common_templates_manifest_url }}/{{ release_tag }}/common-templates-{{ release_tag }}.yaml"
```

Comment:
What about downstream? Will this format work?

Reply:
the release_tag here is assigned in defaults (0.4.0 for example)

Reply:
Sure, but downstream download url might not match the base/release/common-templates-..yml format. And it should not download from upstream sources. So the role should support that case.

Reply:
I don't like the idea of downloading the templates.  They caused issues for us all over the place.   For this release, can you either ship the template in kubevirt-ansible or add the templates as an rpm dependency of kubevirt-ansible?